src/components/tables/DataTable.js

```diff
@@ -115,36 +105,51 @@ export default {
       type: Boolean,
       default: false
     },
-    defaultSort: {
+    selected: {
```

Comment:
I think we should let the selected be controlled under the model and the items be under a prop items, and just sync that. Would be curious to hear others thoughts.

```v-model="yourSelectedRows"```
```:items.sync="tableItems"```